.eslintrc.json

```diff
@@ -1,28 +1,29 @@
 {
-  "parserOptions": {
-    "ecmaVersion": 5
-  },
+  "env": { "commonjs": true },
   "extends": "eslint:recommended",
-  "env": {
-    "commonjs": true
-  },
+  "parserOptions": { "ecmaVersion": 5 },
   "rules": {
-    "strict": [2, "global"],
-    "block-scoped-var": 2,
-    "consistent-return": 2,
-    "eqeqeq": [2, "smart"],
-    "guard-for-in": 2,
-    "no-caller": 2,
-    "no-extend-native": 2,
-    "no-loop-func": 2,
-    "no-new": 2,
-    "no-param-reassign": 2,
-    "no-return-assign": 2,
-    "no-unused-expressions": 2,
-    "no-use-before-define": 2,
-    "radix": [2, "always"],
-    "indent": [2, 2],
-    "quotes": [2, "double"],
-    "semi": [2, "always"]
+    "block-scoped-var": "error",
+    "consistent-return": "error",
+    "eqeqeq": "error",
```

Comment:
The original file has `[2, "smart"]`. Do we need to keep that?

Reply:
It's been overwritten to use the standard, which is a bit stricter, but makes it more explicit when you're choosing to compare against null / undefined.